src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -70,12 +70,25 @@ LegacyConfigForm.propTypes = {
     allowAnonymousPosts: PropTypes.bool.isRequired,
     allowAnonymousPostsPeers: PropTypes.bool.isRequired,
     blackoutDates: PropTypes.string.isRequired,
-  }).isRequired,
+  }),
   onSubmit: PropTypes.func.isRequired,
   // eslint-disable-next-line react/forbid-prop-types
   formRef: PropTypes.object.isRequired,
   intl: intlShape.isRequired,
   title: PropTypes.string.isRequired,
 };
 
+LegacyConfigForm.defaultProps = {
+  appConfig: {
+    divideByCohorts: false,
```

Comment:
can we add `title` default prop here?

Reply:
`title` is required - the fix was to add a title in the test.